devito/ops/node_factory.py

```diff
@@ -0,0 +1,125 @@
+from sympy import Eq, Add, Mul, Pow, Integer, Float, Rational
+from devito.types import Symbol
+
+from devito.ops.utils import namespace
+
+
+class OPSNodeFactory():
+    """
+        Class responsible to generate ops expression for building the OPS ast.
+    """
+
+    def __init__(self):
+        self.ops_access = {}
+
+    def new_symbol(self, name):
+        """
+            Creates a new sympy Symbol object with the given name.
+
+            :param name: Name of the symbol to be created.
```

Comment:
We're switching everything to ``numpydoc``. You can google for it, or take a look at the [reference](https://numpydoc.readthedocs.io/en/latest/format.html) while some examples are available [here](https://github.com/opesci/devito/blob/master/devito/data/decomposition.py)

Reply:
I should have asked about this. Its not a problem, i can configure my editor to use numpydoc as docstring.